src/content/reference/cluster-upgrades/_index.md

```diff
@@ -113,9 +113,11 @@ At the same time we advise to read upfront these [recommendations](#recommendati
 Whenever a new Kubernetes minor version is released by the Kubernetes project, we aim to make that version available in a new major release of our stack within 30 days from the Kubernetes release.
 
 Once we publish a new major release, we deprecate the oldest major release.
-This means that no new clusters can be created using that old release version.
+This means that new clusters with deprecated releases can only be created using `gsctl`.
 Existing clusters, however, are not affected.
 
+Creating clusters with deprecated releases is generally not recommended. A valid use case for doing so would be testing upgrades in a separate cluster.
```

Comment:
The way it is worded no implies that there are other valid use cases. If this is the only valid use case, I recommend using stronger language, since we want to discourage this behavior.

Reply:
WDYT?
```suggestion
Creating clusters with deprecated releases is generally not recommended. Testing tenant cluster upgrades in a separate cluster should be the only use case.
```

Reply:
Fine with me, no strong onion 🧅